### ISSUE TITLE
Code cleanning or In purpose ? 

### ISSUE THREAD

**Issue by LasneF** [siom79/japicmp#101]
hello 

i do not get if it is in purpose , or just a cut copy pattern here :) 

https://github.com/siom79/japicmp/blob/f6558d69a97b78626618f10cf92a300d06f347f7/japicmp/src/main/java/japicmp/model/BinaryCompatibility.java#L385 

here found class should be an interface so no need to test for Constructor and field

unit test are ok without those lines

   ```
// section 13.4.4 of "Java Language Specification" SE7
		for (JApiImplementedInterface implementedInterface : jApiClass.getInterfaces()) {
			if (implementedInterface.getChangeStatus() == JApiChangeStatus.REMOVED) {
				implementedInterface.setBinaryCompatible(false);
				changedIncompatible = true;
			} else if (implementedInterface.getChangeStatus() == JApiChangeStatus.MODIFIED || implementedInterface.getChangeStatus() == JApiChangeStatus.UNCHANGED) {
				JApiClass foundClass = classMap.get(implementedInterface.getFullyQualifiedName());

===>Interface				if (foundClass != null) {
					if (methodsHaveChangedIncompatible(foundClass, classMap)) {
						implementedInterface.setBinaryCompatible(false);
						changedIncompatible = true;
					}
=====>	                       if (constructorsHaveChangedIncompatible(foundClass, classMap)) {
						implementedInterface.setBinaryCompatible(false);
						changedIncompatible = true;
					}
=====>		if (fieldsHaveChangedIncompatible(foundClass, classMap)) {
						implementedInterface.setBinaryCompatible(false);
						changedIncompatible = true;
					}
				}
				if (implementedInterface.getChangeStatus() == JApiChangeStatus.MODIFIED) {
					implementedInterface.setBinaryCompatible(false);
					changedIncompatible = true;
				}
			}
		}
		return changedIncompatible;
```

as we deals with interface here there should be no

**Comment by LasneF** [comment]
correction , the fields test is valid even if no unit test as it may have static fields 

only the constructor lines can be removed

**Comment by siom79** [comment]
Added unit test for field modifications of interfaces and removed check for changed constructors of interfaces.

**Comment by siom79** [comment]
Released with version 0.7.0.
